R/input-slider.R

```diff
@@ -296,6 +298,37 @@ findStepSize <- function(min, max, step) {
   }
 }
 
+# Throw a warning if ever `value` is not in the [`min`, `max`] range
+validate_slider_value <- function(min, max, value, fun) {
+  if (is.null(min)   || is_na(min) ||
+      is.null(max)   || is_na(max) ||
+      is.null(value) || is_na(value))
+  {
+    stop(call. = FALSE,
+      sprintf("In %s(): `min`, `max`, and `value` cannot be NULL or NA.", fun)
+    )
+  }
+
+  if (!isTRUE(min(value) >= min)) {
+    warning(call. = FALSE,
+      sprintf(
+        "In %s(): `value` should be greater than or equal to `min` (value = %s, min = %s).",
+        fun, paste(value, collapse = ", "), min
+      )
+    )
+  }
+
+  if (!isTRUE(max(value) <= max)) {
```

Comment:
```suggestion
  if (any(max(value) > max)) {
```

(Do not need to be worried about NA values or length 0 values at this point)